app/src/docs/content/dataUnions/frameworkRoles.mdx

```diff
@@ -1,34 +1,17 @@
 # Data Unions Framework Roles
 
-The Data Union framework allows app developers to aggregate and monetise the real-time data their app users create whilst sharing revenue with them. The data from many different users are pooled into products offered on the Streamr Marketplace. 
+Each Data Union has one Admin (owner), and `n` data producing members.
```

Comment:
`n` = arbitrary, numerous, large amount of?

Reply:
Yep, added "(many)" to that